0006-scriptless-scripts-with-ecdsa.adoc

```diff
@@ -0,0 +1,80 @@
+= Scriptless Scripts with ECDSA
+Lloyd Fournier <lloyd@coblox.tech>;
+:toc:
+:revdate: 16-05-2019
+
+NOTE: Author: {authors} +
+Date: {revdate} +
+Issue: https://github.com/comit-network/comit-rs/issues/899
+
+== Context
+
+<<Sanchez18a>> describes a scriptless scripts protocol that can be used to replace HTLCs with conditional payments based on learning a discrete logarithm (rather than a hash pare-image).
+This would make atomic swap indistinguishable from normal transactions (at least in the case where there is no refund transaction).
```

Comment:
```suggestion
This would make atomic swaps indistinguishable from normal transactions (at least in the case where there is no refund transaction).
```